<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFESPIMB040AcquisitionDescription</class>
 <widget class="QWidget" name="QFESPIMB040AcquisitionDescription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>99</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> ac&amp;quisition description </string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="2" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>p&amp;late/experiment:</string>
        </property>
        <property name="buddy">
         <cstring>spinPlate</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="10">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>co&amp;mment:</string>
        </property>
        <property name="buddy">
         <cstring>edtComment</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>sample/&amp;cell:</string>
        </property>
        <property name="buddy">
         <cstring>spinCell</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>c&amp;urrent sample:</string>
        </property>
        <property name="buddy">
         <cstring>edtCurrentSampleName</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QPushButton" name="btnNextCell">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>next</string>
        </property>
       </widget>
      </item>
      <item row="2" column="11" colspan="2">
       <widget class="QFEnhancedLineEdit" name="edtComment">
        <property name="placeholderText">
         <string>comment ...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFEnhancedLineEdit" name="edtCurrentSampleName">
        <property name="placeholderText">
         <string>current sample ...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QSpinBox" name="spinCell">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="spinWell">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QDateTimeEdit" name="edtDateTime">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="14">
       <widget class="QPushButton" name="btnClearAll">
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="11">
       <widget class="QFEnhancedLineEdit" name="edtPrefix">
        <property name="placeholderText">
         <string>prefix ...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>common pre&amp;fix:</string>
        </property>
        <property name="buddy">
         <cstring>edtPrefix</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinPlate">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>date/t&amp;ime:</string>
        </property>
        <property name="buddy">
         <cstring>edtDateTime</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>well:</string>
        </property>
        <property name="buddy">
         <cstring>spinWell</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QPushButton" name="btnFirstCell">
        <property name="text">
         <string>first</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFEnhancedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfenhancedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spinPlate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QFESPIMB040AcquisitionDescription</receiver>
   <slot>spinChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinWell</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QFESPIMB040AcquisitionDescription</receiver>
   <slot>spinChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinCell</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QFESPIMB040AcquisitionDescription</receiver>
   <slot>spinChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>spinChanged()</slot>
 </slots>
</ui>
